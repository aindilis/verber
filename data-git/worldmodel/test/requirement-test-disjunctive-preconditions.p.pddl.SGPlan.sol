
; Time 0.00
; ParsingTime 0.00
; NrActions 1
; MakeSpan
; MetricValue
; PlanningTechnique Modified-FF(enforced hill-climbing search) as the subplanner

0.001: (ACTION1 OBJ1 OBJ2) [1]
