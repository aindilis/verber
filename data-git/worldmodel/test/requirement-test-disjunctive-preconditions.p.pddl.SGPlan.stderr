

Starting SGPlan search ...

Solution found.
Answer file is /var/lib/myfrdcsa/codebases/internal/verber/data/worldmodel/test/requirement-test-disjunctive-preconditions.p.pddl.SGPlan.sol

